jQuery/.eslintrc.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
```

Comment:
Let's move this eslint config to the eslint-config-devextreme repo
We should do it because we want to use the same eslint config across all our jQuery scripts in the future
[Link to the repository](https://github.com/DevExpress/eslint-config-devextreme)

To make it, create a new config file (e.g jQuery.js) in the eslint-config-devextreme repo and just extend it here without overriding any rules :)